### PR TITLE
Mobile Version Contact Section

### PR DESCRIPTION
**Pull request by teke85**
**This Pull Request is based on the Mobile Version Portfolio Project and it fulfils the following objectives.**

**General requirements**
_To make sure that there are no linter errors.
To make sure that correct GitHub Flow is used.
Make sure that work is documented in a professional way._

**Project requirements**

_To include a Contacts Section to the Mobile Version Project.
To add a contact form that guides users' experience in a way that helps them fill in the form without frustration._

### CODE REVIEW

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @teke85, 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- Great job with your design 👏 
- Repository looks professional 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -302,12 +302,25 @@ <h2 class="about-title">About<br />Myself<br /></h2>
...
         <a class="skills--title-link" href="#">
-          <span class="skills--title">Skills</span></a
-        >
+          <span class="skills--title">Skills</span></a>
         <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
       </div>
+      <hr class="underline">
     </section>
 
+    <section class="contact-form-sec">
```
Comment (degisew) [inline index.html:311]: Please try to indent properly.

File: style.css [hunk 2]
```diff
@@ -284,11 +303,11 @@ li {
   color: #6070ff;
 }
 
-/*about section starts here*/
+/* about section starts here */
 
 .about {
   max-width: 375px;
-  margin: 0 auto;
+  margin: 0 auto 133px 0;
```
Comment (degisew) [inline style.css:310]: please try to ignore all the unused codes.

**Review by degisew — COMMENTED** [review]
Hi @teke85 
Great job so far 👏 🎖️ 
 but there are some changes needs change

**### Required changes ♻️ ** 
 check the comments under the review

File: index.html [hunk 3]
```diff
@@ -302,12 +302,25 @@ <h2 class="about-title">About<br />Myself<br /></h2>
...
+    <h1 class="contact-me">Contact me</h1>
+    <p class="contact-paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
```
Comment (scientistsagoemarcus) [inline index.html:313]: I suggest you use any of the(h1-h2) elements for your text. A good job is done congratulations.

Comment (scientistsagoemarcus) [reply index.html:313]: Good work so far.

File: style.css [hunk 4]
```diff
@@ -25,7 +25,7 @@ body {
   max-width: 375px;
   font-family: var(--font-family);
   margin: 0 auto;
```
Comment (AmitKlohan) [inline style.css:27]: Hi @teke85 
 its a good practice to use var to reuse it

**Review by AmitKlohan — COMMENTED** [review]
Hi @teke85 
- Your project looks good

**Comment by scientistsagoemarcus** [comment]
Okay sir

**Comment by scientistsagoemarcus** [comment]
PLEASE this is the folder

**Pull request merged**